man/org.freedesktop.systemd1.xml

```diff
@@ -2768,11 +2768,11 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       readonly t ExecMainStartTimestampMonotonic = ...;
       readonly t ExecMainExitTimestamp = ...;
       readonly t ExecMainExitTimestampMonotonic = ...;
-      readonly t ExecMainHandoverTimestamp = ...;
-      readonly t ExecMainHandoverTimestampMonotonic = ...;
       readonly u ExecMainPID = ...;
       readonly i ExecMainCode = ...;
       readonly i ExecMainStatus = ...;
+      readonly t ExecMainHandoverTimestamp = ...;
+      readonly t ExecMainHandoverTimestampMonotonic = ...;
```

Comment:
"Main" should be dropped here now, no?

Reply:
I think it makes sense to retain, even if it's a different struct? Because it only tracks the main process, intentionally, as discussed. That way it is clear and unambiguous. And if in the future the features Lennart is asking for are added, this public interface needs no change and will stay the same, as more properties for control procs are added.

Reply:
On the other hand, I do not have a preference on the name, so if there's a better one I'm happy to change it - just trying to think in terms of not needing future changes if and when more stuff is added, and to make it clear what it does

Reply:
Hmm, instead, I think the struct field should carry "main".

Reply:
> Hmm, instead, I think the struct field should carry "main_".

fine by me - @keszybz does that work or do you prefer a property rename?

Reply:
Yeah, it WFM.

Reply:
ok, done, thanks